main.tf

```diff
@@ -135,19 +135,13 @@ module "clients" {
   source = "./modules/nomad-cluster"
 
   cluster_name  = "${var.cluster_name}-client"
+  cluster_size  = var.num_servers
```

Comment:
All the tests failed with the error:

```
TestNomadConsulClusterColocatedAmazonLinuxAmi 2019-07-11T03:25:52Z retry.go:72: Check Nomad cluster has expected number of servers and clients
TestNomadConsulClusterColocatedAmazonLinuxAmi 2019-07-11T03:25:52Z nomad_helpers.go:204: Making an HTTP GET to URL http://13.125.155.85:4646/v1/nodes
TestNomadConsulClusterColocatedAmazonLinuxAmi 2019-07-11T03:25:52Z nomad_helpers.go:217: Response from Nomad for URL http://13.125.155.85:4646/v1/nodes: [{"ID":"7bd8d5b9-e3c8-9840-9fa4-16d85e6175d8","Datacenter":"ap-northeast-2c","Name":"i-0ca9666a3502cb0eb","NodeClass":"","Version":"0.7.1","Drain":false,"Status":"ready","StatusDescription":"","CreateIndex":5,"ModifyIndex":7},{"ID":"d5aab2c4-561a-1c35-004c-c0c901cec7d3","Datacenter":"ap-northeast-2a","Name":"i-09b1e8a8734bad989","NodeClass":"","Version":"0.7.1","Drain":false,"Status":"ready","StatusDescription":"","CreateIndex":9,"ModifyIndex":10},{"ID":"e1f0d636-5f8d-2613-da27-2b9c107b2e03","Datacenter":"ap-northeast-2a","Name":"i-0ade4bbf573096367","NodeClass":"","Version":"0.7.1","Drain":false,"Status":"ready","StatusDescription":"","CreateIndex":6,"ModifyIndex":8}]
TestNomadConsulClusterColocatedAmazonLinuxAmi 2019-07-11T03:25:52Z retry.go:84: Check Nomad cluster has expected number of servers and clients returned an error: Expected the cluster to have 6 clients, but found 3. Sleeping for 10s and will try again.
```

I think this was a copy/paste error and this value should be `num_clients`.

Reply:
Yes you are correct ;) Thanks for pointing it out.